numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -3172,22 +3172,20 @@ static npy_hash_t
 static npy_hash_t
 @lname@_arrtype_hash(PyObject *obj)
 {
-    return _Py_HashDouble((double) PyArrayScalar_VAL(obj, @name@));
+    return Npy_HashDouble(obj, (double)PyArrayScalar_VAL(obj, @name@));
 }
 
 /* borrowed from complex_hash */
 static npy_hash_t
 c@lname@_arrtype_hash(PyObject *obj)
 {
     npy_hash_t hashreal, hashimag, combined;
-    hashreal = _Py_HashDouble((double)
-            PyArrayScalar_VAL(obj, C@name@).real);
+    hashreal = Npy_HashDouble(obj, (double)PyArrayScalar_VAL(obj, C@name@).real);
```

Comment:
This is a hair long.

Reply:
yeah, how strict are we about the 79 chars? Lately, I have started to not worry if it I figure the meaning is clear even if you don't see the last few chars.

Happy to change though.

Reply:
We aren't limited by the terminal these days, so we could probably increase the line length as long as readability is good, say 85 characters. Might be worth discussing, especially with C++ in the future, C++ tends to long lines.